sdk/src/main/java/com/mapbox/maps/MapboxMapExt.kt

```diff
@@ -0,0 +1,19 @@
+@file:JvmName("MapboxMapUtils")
+
+package com.mapbox.maps
+
+import com.mapbox.geojson.Point
+import com.mapbox.maps.plugin.delegates.MapCameraManagerDelegate
+
+/**
+ * Calculates geographical coordinates(i.e., longitude-latitude pair) that corresponds
+ * to screen box.
+ *
+ * @param screenBox A box on the screen in \link MapOptions#size platform pixels \endlink.
+ *
+ * @return Returns a batch of geographical coordinates corresponding to the screen coordinates
+ * on the screen.
+ */
+fun MapCameraManagerDelegate.coordinatesForPixels(screenBox: ScreenBox): List<Point> {
+  return this.coordinatesForPixels(listOf(screenBox.min, screenBox.max))
```

Comment:
Yes, but I think we can use this method temporarily and update it when we have the generated method

Reply:
I guess it would be hard to deprecate this method later when the method is directly exposed in MapboxMap, due to the same signature in Kotlin, and it will be breaking for Java users.

Reply:
agree with @pengdev - if we really need it (however I do not see any linked task meaning it's a customer request) - we should add it in gl-native and expose both on Android and iOS at the same time. Otherwise @pengdev 's argument are 💯 correct - adding non-critical method that we're about to deprecate is not wise.